### PR TITLE
did the task

### PR DESCRIPTION
**Pull request by vanyachyzh**
[DEMO LINK](https://vanyachyzh.github.io/react_dynamic-list-of-goods/)

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,27 +1,66 @@
...
-    <button type="button" data-cy="all-button">
-      Load all goods
-    </button>
+  const getAllGoods = async () => {
+    const allGoods = await goodsAPI.getAll();
 
-    <button type="button" data-cy="first-five-button">
-      Load 5 first goods
-    </button>
+    if (allGoods) {
```
Comment (etojeDenys) [inline src/App.tsx:14]: do you really need this if statement?

**Review by etojeDenys — CHANGES_REQUESTED** [review]
add color for `li`'s
image

File: src/App.tsx [hunk 2]
```diff
@@ -1,27 +1,72 @@
...
+
+      <button
+        type="button"
+        data-cy="red-button"
+        onClick={getRedGoods}
+      >
+        Load red goods
+      </button>
+
+      {goods && (<GoodsList goods={goods} />)}
```
Comment (VitaliyBondarenko1982) [inline src/App.tsx:69]:
```suggestion
      {!!goods.length && (<GoodsList goods={goods} />)}
```
or
```suggestion
   <GoodsList goods={goods} />
```

Comment (VitaliyBondarenko1982) [reply src/App.tsx:69]: goods always will be true.
check its length or not check nothing. 
empty array will not be rendered

**Review by VitaliyBondarenko1982 — APPROVED** [review]

**Pull request closed without merging**